src/Makefile.am

```diff
@@ -129,15 +129,17 @@ $(desktop_in_files): %.desktop.in: %.desktop.in.in Makefile
 desktopdir=$(datadir)/applications
 desktop_DATA = $(desktop_in_files:.desktop.in=.desktop)
 
-@INTLTOOL_DESKTOP_RULE@
+$(desktop_DATA): $(addsuffix .in, $(desktop_DATA))
+	cp -f $< $@
 
 component_DATA = kkc.xml
 componentdir = $(datadir)/ibus/component
 
 kkc.xml.in: kkc.xml.in.in
 	$(AM_V_GEN) $(edit) $< > $@.tmp && mv $@.tmp $@
 
-@INTLTOOL_XML_NOMERGE_RULE@
+$(component_DATA): $(addsuffix .in, $(component_DATA))
+	cp -f $< $@
```

Comment:
Similar to the above: kkc.xml.in.in has translatable strings marked as `_longname` etc. We need to use `msgfmt --xml`:
https://www.gnu.org/software/gettext/manual/html_node/msgfmt-Invocation.html#XML-mode-operations
https://www.gnu.org/software/gettext/manual/html_node/Preparing-ITS-Rules.html#Preparing-ITS-Rules